lib/types.ts

```diff
@@ -123,6 +124,19 @@ export type MessageEvent = {
   message: EventMessage;
 } & ReplyableEvent;
 
+/**
+ * Event object for when the user unsends a message in a [group](https://developers.line.biz/en/docs/messaging-api/group-chats/#group)
+ * or [room](https://developers.line.biz/en/docs/messaging-api/group-chats/#room).
+ * [Unsend event](https://developers.line.biz/en/reference/messaging-api/#unsend-event)
+ */
+export type UnsendEvent = {
+  type: "unsend";
+  /**
+   * The message ID of the unsent message
+   */
+  unsend: { messageId: String };
```

Comment:
`string` should be used in this case.

ref: https://stackoverflow.com/a/14727461/605165

Reply:
Thank you!